frontend/src/concepts/projects/accessChecks.tsx

```diff
@@ -34,3 +35,21 @@ export const useProjectPermissionsAccessReview = (
     },
     shouldRunCheck,
   );
+
+export const useProjectPermissionsTabVisible = (
+  projectName: string,
+  shouldRunCheck?: boolean,
+): ReturnType<typeof useAccessReview> =>
+  useAccessReview(
+    {
+      group: 'rbac.authorization.k8s.io',
+      resource: 'rolebindings',
+      namespace: projectName,
+      verb: 'list',
+    },
+    shouldRunCheck,
+  );
+
+// TODO: expand this out to meet future needs
+export const useProjectSettingsTabVisible = (): boolean =>
+  useIsAreaAvailable(SupportedArea.BIAS_METRICS).status;
```

Comment:
Why is this based on BIAS_METRICS?

Reply:
I see below that's what it was based on before but I don't understand the link with project perms

Reply:
Settings tab as no other value today -- but it may in the future, hence the TODO